course-v2/assets/videojs-imports.js

```diff
@@ -2,6 +2,7 @@ import { initVideoTranscriptTrack } from "./js/video_transcript_track"
 import { initPlayBackSpeedButton } from "./js/video_playback_speed"
 import { initVideoFullscreenToggle } from "./js/video_fullscreen_toggle"
 import { initDownloadButton } from "./js/video-download-button"
+import { videoToggleDownloadPopup } from "./js/video_toggle_download"
```

Comment:
Following the existing convention, I think it'd be nice to name this method something like `initVideoDownloadPopup`.

Reply:
Thanks for your suggestion. I have updated the function name along with the file name.